rgw/standalone/extensa/extensa_ms_archive_deploy.yaml

```diff
@@ -0,0 +1,590 @@
+                   args:
+                     placement:
+                       label: mgr
+               - config:
+                   command: apply
+                   service: mon
+                   args:
+                     placement:
+                       label: mon
+
+         ceph-arc:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: add_hosts
+                   service: host
+                   args:
+                     attach_ip_address: true
+                     labels: apply-all-labels
+               - config:
+                   command: apply
+                   service: mgr
+                   args:
+                     placement:
+                       label: mgr
+               - config:
+                   command: apply
+                   service: mon
+                   args:
+                     placement:
+                       label: mon
+       desc: RHCS deploy more mons and mgrs.
+       polarion-id: CEPH-83575222
+       destroy-cluster: false
+       module: test_cephadm.py
+       name: deploy more mons and mgrs
+
+   - test: # deploy osds
+       abort-on-fail: true
+       clusters:
+         ceph-pri:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: apply
+                   service: osd
+                   args:
+                     all-available-devices: true
+         ceph-sec:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: apply
+                   service: osd
+                   args:
+                     all-available-devices: true
+         ceph-arc:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: apply
+                   service: osd
+                   args:
+                     all-available-devices: true
+       desc: RHCS deploy osds.
+       polarion-id: CEPH-83575222
+       destroy-cluster: false
+       module: test_cephadm.py
+
+   - test: # deploy sync rgws
+       abort-on-fail: true
+       clusters:
+         ceph-pri:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: apply
+                   service: rgw
+                   pos_args:
+                     - india.sync
+                   args:
+                     port: 80
+                     placement:
+                       nodes:
+                         - extensa024
+                         - extensa025
+         ceph-sec:
+           config:
+             verify_cluster_health: true
+             steps:
+               - config:
+                   command: apply
+                   service: rgw
+                   pos_args:
+                     - india.sync
```

Comment:
Hi @yuva-gayam 
there is no harm in the same name for rgw service on all the sites, but just for clarity we deploy rgw for primary as pri.sync and for secondary sec.sync
similarly for io and for archive zone as well..

cc: @ckulal